packages/react-data-grid/style/rdg-cell.less

```diff
@@ -131,10 +128,6 @@
     border-color: transparent transparent black transparent;
 }
 
-.react-grid-Canvas.opaque .react-grid-Cell.cell-tooltip:hover .cell-tooltip-text {
```

Comment:
We never set any additional class on `.react-grid-Canvas`, so this whole thing can go.